src/v2/providers/https.ts

```diff
@@ -36,14 +36,6 @@ import * as options from '../options';
 
 export { Request, CallableRequest, FunctionsErrorCode, HttpsError };
 
-export interface HttpsOptions extends Omit<options.GlobalOptions, 'region'> {
```

Comment:
If we go down this path, we need to import & re-export `HttpsOptions` from here so it's not a breaking change.